OpenRA.Editor/Form1.cs

```diff
@@ -238,7 +238,9 @@ void PrepareMapResources(ModData modData, Map map)
 					if (rsi != null && rsi.EditorPalette != null && rsi.EditorPalette.Contains("terrain"))
 						templatePalette = palette;
 
-					var template = RenderUtils.RenderActor(info, tileset, templatePalette);
+					var race = Program.Rules.Actors["world"].Traits.WithInterface<CountryInfo>().First().Race;
```

Comment:
RenderEditorOnly has problems, but if you comment that out then everything else works well.